docs/remoteExams.md

```diff
@@ -6,6 +6,55 @@ This page lists sample assessment configurations for remote exams, where student
 *See the [Access control](accessControl.md) page for details on `allowAccess` rules.*
 
 
+## Exams in the Computer-Based Testing Facility (CBTF)
+
+If you are using the CBTF for remote proctoring then the access control should look like:
+
+```json
+"allowAccess": [
+    {
+        "role": "TA",
+        "credit": 100
+    },
+    {
+        "mode": "Exam",
+        "examUuid": "c48e40db-258d-43c8-bb26-1f559ffe2228",
+        "credit": 100
+    }
+],
+```
+
+Some notes about this configuration:
+
+* The `examUuid` parameter should be copied from the CBTF Scheduler website for the specific exam. Each exam has its own unique `examUuid` and it's vital that the correct value is used for each separate exam.
+* Date restrictions and time limits must not be set for the exam. All limits will be automatically enforced by the CBTF on a per-student basis, taking into account conflict exams and disability accommodations.
+
+If some students need to take the exam outside of the CBTF then they can be added separately, like the following example. See the (next section)[#synchronous-timed-exams] for more details.
+
+```json
+"allowAccess": [
+    {
+        "role": "TA",
+        "credit": 100
+    },
+    {
+        "uids": ["student1@illinois.edu", "student2@illinois.edu"],
+        "mode": "Public",
+        "credit": 100,
+        "startDate": "2020-04-20T11:00:00",
+        "endDate": "2020-04-20T12:40:00",
+        "timeLimitMin": 90,
+        "showClosedAssessment": false
+    },
+    {
+        "mode": "Exam",
+        "examUuid": "c48e40db-258d-43c8-bb26-1f559ffe2228",
+        "credit": 100
+    }
+],
+```
+
```

Comment:
Should we add a note saying that "the additional access rule for specific students can be added after other students already completed the exam using CBTf. This can facilitate setting up later accommodations for students that missed the exam."

Reply:
Should we also say that in this case the order does not matter?

Reply:
done and done